src/System.Text.Json/tests/Utf8JsonReaderTests.cs

```diff
@@ -969,6 +969,9 @@ public static void InvalidJsonSingleSegment(string jsonString, int expectedlineN
         [InlineData("\"h\u6F22\u5B57ello\"//This is a \u6F22\u5B57comment after json with new line\n", "//This is a \u6F22\u5B57comment after json with new line\n", 64)]
         [InlineData("{\"a\u6F22\u5B57ge\" : \n//This is a \u6F22\u5B57comment between key-value pairs\n 30}", "//This is a \u6F22\u5B57comment between key-value pairs\n", 66)]
         [InlineData("{\"a\u6F22\u5B57ge\" : 30//This is a \u6F22\u5B57comment between key-value pairs on the same line\n}", "//This is a \u6F22\u5B57comment between key-value pairs on the same line\n", 84)]
+        [InlineData("\"a\u6F22\u5B57lpha\" \r\n//This is a comment with a carriage return\r//Another single-line comment", "//This is a comment with a carriage return\r", 59)]
+        [InlineData("\"a\u6F22\u5B57lpha\" \r\n//This is a comment with a line break\n//Another single-line comment", "//This is a comment with a line break\n", 54)]
```

Comment:
Can you please add a test where the data is split between the `/r` and `/n`?

Add the test here, PartialJson:
https://github.com/dotnet/corefx/blob/8c5941b480487fc12da21d1bbe7235a8c2b2afcb/src/System.Text.Json/tests/Utf8JsonReaderTests.cs#L619

Also, can you add these tests cases for Skip/SkipSingleSegment/AllowSingleSegment as well?
- https://github.com/dotnet/corefx/blob/8c5941b480487fc12da21d1bbe7235a8c2b2afcb/src/System.Text.Json/tests/Utf8JsonReaderTests.cs#L1171
- https://github.com/dotnet/corefx/blob/8c5941b480487fc12da21d1bbe7235a8c2b2afcb/src/System.Text.Json/tests/Utf8JsonReaderTests.cs#L1231
- https://github.com/dotnet/corefx/blob/8c5941b480487fc12da21d1bbe7235a8c2b2afcb/src/System.Text.Json/tests/Utf8JsonReaderTests.cs#L1056